pep-0546.txt

```diff
@@ -0,0 +1,118 @@
+PEP: 546
+Title: Backport ssl.MemoryBIO and ssl.SSLObject to Python 2.7
+Version: $Revision$
+Last-Modified: $Date$
+Author: Victor Stinner <victor.stinner@gmail.com>,
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 30-May-2017
+
+
+Abstract
+========
+
+Backport ssl.MemoryBIO and ssl.SSLObject classes from Python 3 to Python
+2.7 to enhance the overall security of Python 2.7.
+
+
+Rationale
+=========
+
+While Python 2.7 is getting closer to its end-of-line (scheduled for
+2020), it is still used on production and the Python community is still
+responsible for its security.
```

Comment:
And to facilitate the future adoption of PEP 543, which will improve security for Python3 users.